exercises/scripts/control-plane.sh

```diff
@@ -5,12 +5,19 @@ API_ADV_ADDRESS=$2
 
 kubeadm init --pod-network-cidr $POD_CIDR --apiserver-advertise-address $API_ADV_ADDRESS | tee /vagrant/kubeadm-init.out
 
+echo "KUBELET_EXTRA_ARGS=--node-ip=$API_ADV_ADDRESS --cgroup-driver=systemd" > /etc/default/kubelet
+systemctl restart kubelet
+
 mkdir -p /home/vagrant/.kube
 cp -i /etc/kubernetes/admin.conf /home/vagrant/.kube/config
 chown vagrant:vagrant /home/vagrant/.kube/config
 mkdir -p /root/.kube
 cp -i /etc/kubernetes/admin.conf /root/.kube/config
 
-kubectl apply -f https://docs.projectcalico.org/v3.22/manifests/calico.yaml
```

Comment:
I noticed that yesterday as well. This is still available but at https://projectcalico.docs.tigera.io/v3.22/manifests/calico.yaml. The recommended YAML manifests from the documentation are the ones you used here.

Reply:
Thanks for the link. I used the new way written in Calico documentation. I think it worth keeping it.